### PR TITLE
welcome_run-d-program-locally

### PR DESCRIPTION
**Pull request by wilzbach**

### CODE REVIEW

**Comment by wilzbach** [comment]
Automatically imported from https://github.com/stonemaster/dlang-tour/pull/408, still needs to address feedback

**Comment by wilzbach** [comment]
@UplinkCoder - I moved the translation to its new home at the German repository. Do you have time to finish your review? :)

File: welcome/run-d-program-locally.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# D Programme lokal ausführen
+
+Eine D Installation beinhaltet unter anderem den D Compiler `dmd`,
+ein Skripttool `rdmd` und den Packetmanager `dub`.
+
+### DMD Compiler
+
+Der *DMD* Compiler kompiliert D Dateien und erstellt eine Binärdatei.
+Auf der Kommandozeile kann *DMD* mit dem Dateinamen aufgerufen werden:
```
Comment (WebFreak001) [inline welcome/run-d-program-locally.md:9]: idk but `In der Kommandozeile` feels more right

File: welcome/run-d-program-locally.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+in einer D-Quelldatei mit Ausführberechtigungen eingefügt werden und erlaubt eine
+Ausführung der Datei analog zu Skriptsprachen.
+
+Weitere Informationen sind in der [Online-Dokumentation](https://dlang.org/rdmd.html)
+oder per `rdmd --help` zu finden.
+
+### Paketmanager `dub`
+
+Der Standardpaketmanager für D ist [`dub`](http://code.dlang.org). Wenn `dub` lokal
+installiert ist, kann ein neues Project `hello` mit folgenden Kommando angelegt
```
Comment (WebFreak001) [inline welcome/run-d-program-locally.md]: Projekt instead of Project

File: welcome/run-d-program-locally.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    rdmd hallo.d
+
+Auf UNIX System kann das Shebangkommando `#!/usr/bin/env rdmd` als erste Zeile
+in einer D-Quelldatei mit Ausführberechtigungen eingefügt werden und erlaubt eine
+Ausführung der Datei analog zu Skriptsprachen.
+
+Weitere Informationen sind in der [Online-Dokumentation](https://dlang.org/rdmd.html)
+oder per `rdmd --help` zu finden.
+
+### Paketmanager `dub`
```
Comment (WebFreak001) [inline welcome/run-d-program-locally.md:32]: Once you call it Packetmanager and then 3 times you call it Paketmanager. This should be more consistent. I would go with `Packetmanager` on all of them

File: welcome/run-d-program-locally.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+Wenn der Paketmanager per `dub` innerhalb eines Projektes ausgeführt wird,
+werden alle Abhängigkeiten des Projektes herunterladen, kompiliert, gelinkt
+und anschließend wird die Anwendung ausgeführt.
+Das Kommando `dub build` verhält sich ähnlich, aber führt die Anwendung nicht aus.
+
+Weitere Informationen sind in der [Online-Dokumentation](https://code.dlang.org/docs/commandline)
+oder per `dub --help` zu finden.
+
+All vorhandene `dub` Pakete können durch das [Web portal](https://code.dlang.org)
```
Comment (WebFreak001) [inline welcome/run-d-program-locally.md]: Alle

Comment (WebFreak001) [inline welcome/run-d-program-locally.md]: I would use Web-Portal (also with a capital P)

**Comment by WebFreak001** [comment]
just flew over a bit, didn't go as in-depth with the PRs I'm assigned to. So it would be great if the people who were actually going to correct it also can look over it

File: welcome/run-d-program-locally.md (hunk 3 above)
Comment (wilzbach) [reply welcome/run-d-program-locally.md:32]: > I would go with Packetmanager on all of them

Good catch, while it's not part of the Duden I would go with "Paketmanager" as the Wikipedia page is also about "Paketverwaltung":

https://de.wikipedia.org/wiki/Paketverwaltung

**Comment by wilzbach** [comment]
Just merging this as it always can be improved gradually

**Pull request merged**